python3-needly.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "python3-needly",
+    "buildsystem": "simple",
+    "build-commands": [
+        "pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"needly\" --no-build-isolation",
+        "mkdir -pv /app/share/applications/",
+        "cp -v *.desktop /app/share/applications/",
+        "mkdir -pv /app/share/icons/hicolor/128x128/apps/",
+        "cp -v *.png /app/share/icons/hicolor/128x128/apps/",
+        "mkdir -pv /app/share/appdata/",
+        "cp -v *.desktop /app/share/appdata"
```

Comment:
You should use the explicit file name not a glob. And using `install -D` is still less commands and clearer.

Reply:
I tried to change this to `install -D` and stopped using globs.